mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+  /**
+   * Checkbox status
+   */
+  checked: boolean;
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Checkbox`s value
+   */
+  value?: string;
+  /**
+   * Checkbox's name
+   */
+  name?: string;
+  /**
+   * The flag sets the element to be required
+   */
+  required?: boolean;
+  /**
+   * Callback triggered on change checkbox
+   */
+  onChange: () => void;
+}
+
+export const Checkbox: React.FC<CheckboxProps> = (props : CheckboxProps) => {
```

Comment:
Add js-doc to all publicmembers

Reply:
added

Reply:
@Ekaterina1994 why do you ask to add checkbox? Does eslint work? @Eremor check eslint too, docs should be required by eslint. if it is not - please update.

Reply:
Elint works/. It's old code/ Now jsdoc exist